tests/translation/test_translation.h

```diff
@@ -596,6 +596,42 @@ test_translation_stringset_multiframe_impl0()
         appendValue(source, str);
 
 
```

Comment:
there is double new-line. Was there already, but you can as well just remove it here.

Reply:
hm, separate white space changes from functional ones, though, or not?